src/Chapter1.hs

```diff
@@ -418,7 +418,7 @@ task is to specify the type of this function.
 -}
 
 squareSum x y = (x + y) * (x + y)
-
+squareSum :: Integer -> Integer -> Integer
```

Comment:
That's correct!

In Haskell, the type signature usually goes before the function body 🙂